LayoutTests/fast/events/mouse-events-on-node-deletion-expected.txt

```diff
@@ -0,0 +1,20 @@
+Verifies that mouse events are not fired on deleted nodes
+
+On success, you will see a series of "PASS" messages, followed by "TEST COMPLETE".
+
+
+outer received mouseover
```

Comment:
This isn't the way js-test.js tests are typically written.
We should have a bunch of `shouldBe` function calls validating the outcome.
The current output doesn't make it clear when the test is passing
and when it's failing so we should update the test to do that.